### ISSUE TITLE
Clickable and non clickable items, in ComboBox dropdown ,are not differentiable on high contrast mode

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#13771]
Clickable and non clickable items, in ComboBox dropdown ,are not differentiable on high contrast mode

### Environment Information

OS : 10 
Browser: Edge

### Describe the issue:

Clickable and non clickable items,in ComboBox ,must be differentiated easily on high contrast mode.Clickable and non clickable items are not differentiable in high contrast mode on office ui fabric because they are of same color. There should be more clear color differentiation.

### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox
![cliackable_not_clickable](https://user-images.githubusercontent.com/62233982/85517011-76ff3e00-b61c-11ea-8f0a-80249fd27079.PNG)

#### Actual behavior:
Observed that clickable and non clickable items are not differentiable in high contrast mode on office ui fabric.

#### Expected behavior:
clickable and non clickable items should  be  differentiable in high contrast mode on office ui fabric.

**Comment by paulgildea** [comment]
@xugao adding to the #11742 list.

**Comment by xugao** [comment]
@v-ajrai, correct me if i'm wrong. in this case user can tell if it's clickable by hovering over each item. Also note, HC mode has limited set of color to pick from.

**Comment by v-ajrai** [comment]
@xugao ,Yes i agree on your point but can't there be any mechanism to differentiate between these clickable and non clickable items ,like-we can increase contrast or boldness of the non clickable items,if possible, because end user  is expecting some more clear differentiation visually.

**Comment by xugao** [comment]
@betrue-final-final  - the only color i can think of giving the header section is "GreyText" (disabled color) . what do you think we should do here?

**Comment by betrue-final-final** [comment]
I was going to suggest "GrayText", so I concur.

**Comment by v-ajrai** [comment]
@xugao .I  think it should full fill the purpose
